src/test/java/BaseTest.java

```diff
@@ -35,10 +35,10 @@ public static void closeBrowser(){
         LoginTests.driver.quit();
     }
 
-//    protected static void navigateToPage() {
-//        String url = "https://bbb.testpro.io/";
-//        driver.get(url);
-//    }
+    protected static void navigateToPage() {
+    String url = "https://bbb.testpro.io/";
+    driver.get(url);
+    }
```

Comment:
Please finish the homework 15!

Homework #15:
Java Git

1. Create your free GitHub Account.

2. Finish all the setup steps. 

3. Clone a repository.

4. In the first test change URL from "bbb.testpro.io" to "testpro.io".

5. Commit your changes to a new branch.

6. Push your code to a remote repository.

For example

![image](https://user-images.githubusercontent.com/104761482/211666834-05a7b8d3-6fe2-4fe2-ae86-a6814290e69a.png)

**4. In the first test change URL from "bbb.testpro.io" to "testpro.io".**

![image](https://user-images.githubusercontent.com/104761482/211666927-633623a0-862b-4c5b-a8ba-b42066ec13d3.png)

5. Commit your changes to a new branch.

6. Push your code to a remote repository.